content/docs/other/index.md

```diff
@@ -3,3 +3,7 @@ title: Other Documentation
 ---
 
 
+
+* [A word on backward compatibility](/docs/other/a_word_on_backward_compatibility.html)  
```

Comment:
We've used relative links on other pages, and I'm not sure if we have a strong opinion about absolute vs relative. I mostly care about consistency, and we can also make them uniform later.